releases/v0.13.4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.4
+name: 0.13.4
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: a3eee257fc83022342b49c0b959a17d29457f33e
```

Comment:
Shouldn't this be https://github.com/submariner-io/shipyard/commit/32d409523c0f5ac0316650048e410284d0f1e2d5?

Reply:
I changed it to this, which I think is the tip of the release-0.13 branch.

Reply:
Sorry, I was looking at the wrong branch!